pages/agent/v3.md

```diff
@@ -135,6 +135,37 @@ When the experiment is enabled, the agent will use different lock files from age
 >🛠 Experimental feature
 > To use flock file locks, set <code>experiment="flock-file-locks"</code> in your <a href="/docs/agent/v3/configuration#experiment"> agent configuration</a>.
 
+### Kubernetes exec
+
+Modifies `start` and `bootstrap` in such a way that they can run in separate Kubernetes containers in the same pod.
+
+Currently, this experiment is being used by [agent-stack-k8s](https://github.com/buildkite/agent-stack-k8s).
+
+This will result in errors unless orchestrated in a similar manner to that project. Please see the [README](https://github.com/buildkite/agent-stack-k8s/blob/main/README.md) of that repository for more details.
+
+
+>🛠 Experimental feature
+> To use flock file locks, set <code>experiment="kubernetes-exec"</code> in your <a href="/docs/agent/v3/configuration#experiment"> agent configuration</a>.
+
+### Job API
+
+Exposes a local API for the currently running job to introspect and mutate its state in the form of environment variables. This allows you to write scripts, hooks and plugins in languages other than bash, using them to interact with the agent.
+
+The API is exposed via a Unix Domain Socket, whose path is exposed to running jobs with the `BUILDKITE_AGENT_JOB_API_SOCKET` environment variable, and authenticated with a token exposed using the `BUILDKITE_AGENT_JOB_API_TOKEN` environment variable, using the `Bearer` HTTP Authorization scheme.
+
+The API exposes the following endpoints:
+
+- `GET /api/current-job/v0/env` - returns a JSON object of all environment variables for the current job
+- `PATCH /api/current-job/v0/env` - accepts a JSON object of environment variables to set for the current job
+- `DELETE /api/current-job/v0/env` - accepts a JSON array of environment variable names to unset for the current job
+
```

Comment:
the formatting of this is getting mangled in render:
<img width="755" alt="CleanShot 2023-03-17 at 15 38 30@2x" src="https://user-images.githubusercontent.com/15753101/225798352-ef713a4f-de78-4d71-8cff-514bf971421c.png">

@mbelton-buildkite do you know why that might be?

Reply:
Hmm not sure what caused that. Are you still seeing it that way? The list looks okay when I open [the page on Render](https://bk-docs-pr-1954.onrender.com/docs/agent/v3):

<img width="727" alt="Screenshot 2023-03-17 at 4 28 18 pm" src="https://user-images.githubusercontent.com/119824349/225820310-a0233db8-b8f3-40cc-9426-a3e0ba22ad97.png">

Reply:
oh, i think i might've made that screenshot after i added a newline before the list, but maybe before the render preview compiled. no worries!